openlibrary/templates/type/edition/view.html

```diff
@@ -524,33 +523,29 @@ <h4>$:_('Links <span class="gray small sansserif">outside Open Library</span>')<
         $:render_template("observations/review_component", work, reader_observations, page.key)
         $ component_times['Review component'] = time() - component_times['Review component']
       <a id="lists-section" class="section-anchor"></a>
-      <div class="workFooter tab-section">
-        $# pages like /books/ia:foo00bar are fake records created from metadata API.
-        $# Adding them to lists doesn't work. Disabling it to avoid any issues.
-        $if "lists" in ctx.features and not page.is_fake_record():
+      $# pages like /books/ia:foo00bar are fake records created from metadata API.
+      $# Adding them to lists doesn't work. Disabling it to avoid any issues.
+      $if "lists" in ctx.features and not page.is_fake_record():
```

Comment:
I moved this conditional such that it wraps the entire "Lists" section div.  This was done to avoid rendering an entirely empty lists section on "fake record" pages (as seen below):

![Screenshot 2025-04-01 155213](https://github.com/user-attachments/assets/8ad9e4e0-87c6-4a72-9099-4032a07f891a)
_The border of the "Lists" section is present when the `if` statement is nested within the div_